code/datums/mutations/body.dm

```diff
@@ -173,18 +173,30 @@
 /datum/mutation/human/race
 	name = "Monkified"
 	desc = "A strange genome, believing to be what differentiates monkeys from humans."
+	text_gain_indication = "You feel unusually monkey-like."
+	text_lose_indication = "You feel like your old self."
 	quality = NEGATIVE
 	time_coeff = 2
 	locked = TRUE //Species specific, keep out of actual gene pool
+	var/datum/species/original_species = /datum/species/human
 
 /datum/mutation/human/race/on_acquiring(mob/living/carbon/human/owner)
 	if(..())
 		return
-	. = owner.monkeyize()
+	if(!ismonkey(owner))
+		original_species = owner.dna.species.type
+		owner.Immobilize(2 SECONDS)
```

Comment:
Feel free to remove this line here and that one few down there after switching back to monkeyze/humanize.